docs/packages/transformers.md

```diff
@@ -98,6 +98,18 @@ Alternatively, you can use the [`transformerMetaHighlight`](#transformermetahigh
 
 ---
 
+### `transformerNotationWordHighlight`
+
+Use `[!code highlight[word]]` to highlight a word (adding `highlighted-word` class).
```

Comment:
About the syntax, maybe `[!code word:Hello]`? 

- Use `word:` instead of `highlight:` to avoid confusion of the line highlight.
- Use `:` instead of `[]` to make it similar to match with regex.

And should it be global to the code snippet, or only that line?

Reply:
I initially used `[!code highlight[word]]` as it is more straightforward and intuitive than `word:word`, and it doesn't conflict with our line highlight regex.

It is not global (same as line highlight), you can highlight the same word in multiple lines with `[!code highlight[word]:2]`

Reply:
> `word:word`

Well, you will have `highlight[highlight]` as well 🤣. I still prefer to not call it `highlight` as it's already been using in the line highlight

> It is not global

I wonder if it make more sense to have it global? Otherwise it might be tricky if you want to highlight a lot same token, or highlight multiple different tokens on the same line. I imagine it could be:

```ts
// [!code word:hello] // highlight all "hello"
let hello = 'hello world'
console.log(hello)
```

```ts
// [!code word:hello:2] // highlight "hello" up to 2 occurrence
let hello = 'hello world'
console.log(hello) // so not this
```


```ts
let hello = 'hello world'
// [!code word:hello:1] // highlight "hello" up to 1 occurrence
console.log(hello) // so only this, counting from the notation
```

Reply:
Yeah that makes sense

Reply:
>Well, you will have highlight[highlight] as well 🤣. I still prefer to not call it highlight as it's already been using in the line highlight

I have to clarify I meant it is easier for people to associate "word highlight" with the syntax of `highlight[<word>]` than `word:<word>`.

anyway let's use `word:word` as a conclusion